src/client/eppo-client-assignment-details.spec.ts

```diff
@@ -159,26 +165,30 @@ describe('EppoClient get*AssignmentDetails', () => {
       },
       matchedAllocation: {
         key: 'experiment',
+        name: 'Allocation for experiment',
         allocationEvaluationCode: AllocationEvaluationCode.MATCH,
-        orderPosition: 2,
+        orderPosition: 3,
       },
       unmatchedAllocations: [
         {
           key: 'id rule',
+          name: 'Allocation for id rule',
           allocationEvaluationCode: AllocationEvaluationCode.FAILING_RULE,
-          orderPosition: 0,
+          orderPosition: 1,
         },
         {
           key: 'internal users',
+          name: 'Allocation for internal users',
```

Comment:
If you would like for the test data here to more closely match how `name` and `key` typically look for allocations, it would look like

```suggestion
          key: 'allocation-987',
          name: 'Internal Users',
```

where "987" would be a unique integer reference to the "Internal Users" allocation

Reply:
just a heads up that these keys need to match the `ufc/flags-v1.json`. It looks like this key has existed for some time: https://github.com/Eppo-exp/sdk-test-data/pull/10/files. Happy to do another pass and revamp our test data, but I'm not sure it's in scope of this PR.

Reply:
Ah okay, gotcha. Let's keep as-is in that case